db/changes.go

```diff
@@ -424,6 +424,7 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 			oldestSkipped := db.changeCache.getOldestSkippedSequence()
 			if oldestSkipped > 0 {
 				lowSequence = oldestSkipped - 1
+				base.Infof(base.KeyChanges, "%d is the oldest skipped sequence, using stable sequence number of %d for this feed", oldestSkipped, lowSequence)
```

Comment:
Should add user to this logging (although it would be a candidate for handler/blip context in #3815).

Reply:
Agreed, will add user for now.